docker-compose.test.ct.yaml

```diff
@@ -0,0 +1,226 @@
+      - cray-smd
+    entrypoint: entrypoint.sh functional -c /src/app/tavern_global_config_ct_test.yaml -p /src/app
+    networks:
+      - fas
+  ct-tests-smoke:
+    build:
+      context: test/ct/
+      dockerfile: Dockerfile
+    depends_on:
+      - cray-fas
+    entrypoint: entrypoint.sh smoke -f smoke.json -u http://cray-fas:28800
+    networks:
+      - fas
+  etcd:
+    image: artifactory.algol60.net/quay.io/coreos/etcd:v3.4.7
+    environment:
+      - ALLOW_NONE_AUTHENTICATION=yes
+      - ETCD_ADVERTISE_CLIENT_URLS=http://etcd:2379
+      - ETCD_LISTEN_CLIENT_URLS=http://0.0.0.0:2379
+    networks:
+      - fas
+  cray-fas:
+    build:
+      context: .
+      dockerfile: Dockerfile.integration.Dockerfile
+    environment:
+      - SMS_SERVER=http://cray-smd:27779
+      - CRAY_VAULT_AUTH_PATH=auth/token/create
+      - CRAY_VAULT_ROLE_FILE=configs/namespace
+      - CRAY_VAULT_JWT_FILE=configs/token
+      - S3_ENDPOINT=http://s3
+      - VAULT_ADDR=http://vault:8200
+      - VAULT_TOKEN=hms
+      - VAULT_KEYPATH=hms-creds
+      - VAULT_SKIP_VERIFY=true
+      - VAULT_ENABLED=true
+      - LOG_LEVEL=DEBUG
+      - TRS_IMPLEMENTATION=LOCAL
+      - HSMLOCK_ENABLED=true
+      - STORAGE=ETCD
+      - ETCD_HOST=etcd
+      - ETCD_PORT=2379
+    networks:
+      - fas
+    depends_on:
+      - etcd
+      - s3
+      - cray-smd
+      - cray-hms-hmcollector
+      - redfish-simulator
+      - vault-kv-enabler
+  s3:
+    image: artifactory.algol60.net/docker.io/library/nginx:1.18.0
+    hostname: s3
+    volumes:
+      - ./Images:/usr/share/nginx/html
+#    ports:
+#      - "9000:80"
+    networks:
+      - fas
+  vault:
+    hostname: vault
+    image: artifactory.algol60.net/docker.io/library/vault:1.5.5
+    environment:
+      - VAULT_DEV_ROOT_TOKEN_ID=hms
+      - VAULT_DEV_LISTEN_ADDRESS=0.0.0.0:8200
+      - VAULT_ADDR=http://127.0.0.1:8200
+    cap_add:
+      - IPC_LOCK
+    networks:
+      - fas
+  vault-kv-enabler:
+    image: artifactory.algol60.net/csm-docker/stable/vault-kv-enabler:1.13.4
+    environment:
+      - VAULT_ADDR=http://vault:8200
+      - VAULT_TOKEN=hms
+      - KV_STORES=hms-creds
+    depends_on:
+      - vault
+    networks:
+      - fas
+  redis:
+    image: artifactory.algol60.net/docker.io/library/redis:5.0-alpine3.12
+    hostname: hms-redfish-redis
+    networks:
+      - fas
+  hmsds-postgres:
+    hostname: hmsds-postgres
+    image: artifactory.algol60.net/docker.io/library/postgres:11-alpine
+    environment:
+      - POSTGRES_PASSWORD=hmsdsuser
+      - POSTGRES_USER=hmsdsuser
+      - POSTGRES_DB=hmsds
+    networks:
+      - fas
+  cray-smd-init:
+    image: artifactory.algol60.net/csm-docker/stable/cray-smd:1.30.9
+    environment:
+      - SMD_DBHOST=hmsds-postgres
+      - SMD_DBPORT=5432
+    depends_on:
+      - hmsds-postgres
+    networks:
+      - fas
+    command: /entrypoint.sh smd-init
+  cray-smd:
+    image: artifactory.algol60.net/csm-docker/stable/cray-smd:1.30.9
```

Comment:
Not a huge issue since we're just using HSM to get /State/Components and feed them to the FAS tests, but it would be nice to set this to the latest stable version of HSM since this one is over 6 months old.

Reply:
updated all to 1.48.0